src/node/hooks/express/static.js

```diff
@@ -9,11 +9,11 @@ exports.expressCreateServer = function (hook_name, args, cb) {
 
   // Cache both minified and static.
   var assetCache = new CachingMiddleware;
-  args.app.all('/(javascripts|static)/*', assetCache.handle);
+  args.app.all(/\/(javascripts|static)\/(.*)/, assetCache.handle);
 
   // Minify will serve static files compressed (minify enabled). It also has
   // file-specific hacks for ace/require-kernel/etc.
-  args.app.all('/static/:filename(*)', minify.minify);
+  args.app.all('/static/:filename', minify.minify);
```

Comment:
`'/static/:filename'` translates under the hood to `/^\/static\/(?:([^\/]+?))\/?$/i`, should this have slashes in it? If so, it should be `'/static/:filename+'`